src/pages/positions-list/index.tsx

```diff
@@ -60,6 +67,7 @@ export const PositionsList = () => {
           }}
           columns={tableColumns}
           data={data || []}
+          onRowClicked={handleRowClick}
```

Comment:
Can you add the prop `customStyles` with this values ```const customStyles = {
  rows: {
    style: {
      cursor: 'pointer',
    },
  },
}```  ? , so the user see the row is clickable